sormas-api/src/main/resources/captions.properties

```diff
@@ -510,6 +512,8 @@ CasePreviousHospitalization.healthFacility=Hospital
 CasePreviousHospitalization.healthFacilityDetails=Hospital name & description
 CasePreviousHospitalization.isolated=Isolation
 CasePreviousHospitalization.prevHospPeriod=Period of hospitalization
+CasePreviousHospitalization.hospitalizationReason=Reason for hospitalization
+CasePreviousHospitalization.otherHospitalizationReason=Specify other reason for hospitalization
```

Comment:
Change to "Specify reason", otherwise the caption will get too long (especially in languages like German)